compiler/bootstrap/compilation/x64/32/Holmakefile

```diff
@@ -14,7 +14,10 @@ README.md: $(CAKEMLDIR)/developers/readme_gen readmePrefix $(patsubst %,%readmeP
 
 cake.S: *$(ARCH)BootstrapScript.sml
 
-cake-$(ARCH)-$(WORD_SIZE).tar.gz: cake.S basis_ffi.c Makefile
-	tar -chzf $@ --transform='s|^|cake-$(ARCH)-$(WORD_SIZE)/|' cake.S basis_ffi.c Makefile
+cake-$(ARCH)-$(WORD_SIZE).tar.gz: cake.S basis_ffi.c ../Makefile
+	cp $(CAKEMLDIR)/how-to.md how-to.md
+	cp ../Makefile Makefile
```

Comment:
I think these should use `$(CP)` for copying. But I also think you shouldn't have to copy these just for the sake of `tar`. You should instead use `tar`'s `--directory` (`-C`) option, e.g., 
```
tar [...] basis_ffi.c --directory .. Makefile --directory $(protect $(CAKEMLDIR)) how-to.md
```

(If they weren't being immediately deleted, they should be done via separate targets instead, e.g.,
```
how-to.md: $(CAKEMLDIR)/how-to.md
    $(CP) $< $@
```
)

Reply:
I see that this was solved yet another way for `basis_ffi.c`: by using a symlink committed to the repository. I am tempted to use this solution rather than `-C`, for consistency and to rely less on `tar` options.